config/build.ts

```diff
@@ -40,31 +39,50 @@ const defaultApiSets = {
 };
 
 
-(() => {
-    Promise.resolve()
-        .then(processSnippets)
+(async() => {
```

Comment:
Hmmm. I already merged this to prod. We thought we had your permission to do that while you were gone. My new manager (Reeza) wanted me to do it right away, so I just did. It did not occur to me that code that isn't ready would be in master. I accepted that PR at Dan Saunders request.